pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -55,6 +56,7 @@ const (
 	certRequestSuffix                     = "-primary-cert-bundle"
 )
 
+var fedramp = os.Getenv("FEDRAMP") == "true"
```

Comment:
Can we add a `log.Info` message for the presence of this var? Also, should "FEDRAMP" be a const?

Reply:
i tried defining it as a const but that threw an error: `os.Getenv("FEDRAMP") == "true" (untyped bool value) is not constant`